.github/workflows/pull-request-reviewed.yaml

```diff
@@ -9,20 +9,15 @@ jobs:
   add-waiting-response:
     if: github.event.review.state != 'approved' && github.actor != github.event.pull_request.user.login
     runs-on: ubuntu-latest
-    permissions:
-      pull-requests: write
```

Comment:
we need to explicitly define the permissions being used here, since the default token will be changing to read shortly